.github/workflows/lint.yml

```diff
@@ -1,17 +1,23 @@
-name: Lint
+name: Lint and Typescript
 
 on: pull_request
 
 jobs:
-  lint:
-    name: lint
+  setup:
+    name: setup
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
This is interesting, it makes it so all three commands are run, even if one of them fails. That way we always get the full lint/prettier/eslint output